tests/test_cif_io.py

```diff
@@ -177,4 +177,21 @@ def mockreturn(*args, **kwargs):
         assert actual == expected
         # assert True
 
-# End of file.
+
+mpcontribs_tst = [
+    ('mpc', 'This function is not yet implemented'),
+    ('fs', None),
+    (False, 'mpc_client must be either mpc or fs')
+]
+
```

Comment:
remove empty lines here.  At some point soon, please take a moment to read, or reread PEP8  which addresses things like this, but in this case, it is easer to see that this list is inputs to this test function if they are not separated by blank lines.